src/hex/mod.rs

```diff
@@ -1086,3 +1097,13 @@ impl Hex {
         self.to_local(range)
     }
 }
+
+impl Debug for Hex {
```

Comment:
```suggestion
#[cfg(not(target_arch = "spirv"))]
impl Debug for Hex {
```

Reply:
Interesting; why is this needed?

Reply:
Deubg and Hash are not supported on spirv IIRC, also on the GPU we don't care much about `Debug`